keyboards/sanctified/dystopia/info.json

```
@@ -0,0 +1,84 @@
+{
+    "keyboard_name": "Dystopia",
+    "url": "",
+    "maintainer": "Sanctified",
+    "width": 17.62,
+    "height": 5.716792398988211,
+    "layouts": {
+        "LAYOUT": {
```

This needs to match the layout macro: 
```suggestion
        "LAYOUT_split_bs": {
```